.env.example

```
@@ -0,0 +1,4 @@
+# PostgreSQL
+POSTGRES_USER=<CHANGE_ME>
+POSTGRES_PASSWORD=<CHANGE_ME>
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The POSTGRES_PASSWORD key should go before the POSTGRES_USER key

(UnorderedKey)